pycbc/inference/models/relbin.py

```diff
@@ -541,10 +556,17 @@ def _loglr(self):
             # with detector response. Otherwise, skip detector response.
 
             if self.still_needs_det_response:
+
+                # TODO: Check that p['tc'] is the appropriate variable here
```

Comment:
Why is there a TODO? Do you want to implement it in the future?

Reply:
I forgot to remove it :D Will edit now. 

Am changing the required params in the next push as well (tested and is working)

Reply:
@ConWea Your new modification looks good.